rca/people/models.py

```diff
@@ -226,8 +226,11 @@ def related_students_cache_key(self):
     def fetch_related_students(self):
         value = []
         if self.legacy_staff_id:
-            value = pull_related_students(self.legacy_staff_id)
-            cache.set(self.related_students_cache_key, value, None)
+            try:
+                value = pull_related_students(self.legacy_staff_id)
+                cache.set(self.related_students_cache_key, value, None)
+            except CantPullFromRcaApi:
+                pass
```

Comment:
- If a StaffPage is added with a bad legacy_staff_id, E.G `0` the api fetch responds with a 404, this was causing the page to break.